### PR TITLE
New year items

### PR DESCRIPTION
**Pull request by Elephoof**

### CODE REVIEW

File: common/data/data.json [hunk 1]
```diff
@@ -515,6 +551,24 @@
         "sell_price": 1024,
         "rarity": 99999,
         "sticky": false
+      },
+      {
+        "item": "Gold Stripped Party Hat",
```
Comment (Iapetus-11) [inline common/data/data.json]: Striped not stripped

**Review by Iapetus-11 — CHANGES_REQUESTED** [review]

File: common/data/data.json [hunk 2]
```diff
@@ -423,7 +423,7 @@
         "sticky": false
       },
       {
-        "item": "Gold Stripped Party Hat",
+        "item": "Gold Striped Party Hat",
```
Comment (Iapetus-11) [inline common/data/data.json]: Also I just realized this should just be Party Hat

**Review by Iapetus-11 — CHANGES_REQUESTED** [review]

File: common/data/data.json (hunk 2 above)
Comment (Elephoof) [reply common/data/data.json]: I thought of making it more new year themed

File: common/data/data.json [hunk 3]
```diff
@@ -409,6 +409,42 @@
...
+        "sticky": false
+      },
+      {
+        "item": "Fireworks Bundle",
+        "sell_price": 1024,
+        "rarity": 8192,
+        "sticky": false
+      },
+      {
+        "item": "New Years Ballon",
```
Comment (Iapetus-11) [inline common/data/data.json]: Balloon

**Review by Iapetus-11 — CHANGES_REQUESTED** [review]

**Review by Iapetus-11 — APPROVED** [review]
sure, thanks

**Pull request merged**